hack/test-external.sh

```diff
@@ -2,19 +2,28 @@
 
 set -e -x -u
 
+minikube tunnel &> /dev/null &
+
 time kapp app-group deploy -y -g gitops -d examples/gitops/
 time kapp app-group delete -y -g gitops
 
-time kapp deploy -y -a istio -f examples/istio-v1.4.0/istio.yml
+time kapp deploy -y -a istio -f examples/istio-v1.12.2/
 time kapp delete -y -a istio
 
-time kapp deploy -y -a cert-manager -f examples/cert-manager-v0.15.1/
+time kapp deploy -y -a cert-manager -f examples/cert-manager-v1.6.1/
 time kapp delete -y -a cert-manager
 
-time kapp deploy -y -a knative -f examples/knative-v0.15.0/
+time kapp deploy -y -a knative -f examples/knative-v1.1.0/
 time kapp delete -y -a knative
 
-time kapp deploy -y -a cf -f examples/cf-for-k8s-v0.2.0-custom/
-time kapp delete -y -a cf
+# TODO Add cf-for-k8s-v5.4.3
+
+time kapp deploy -y -a gk -f examples/gatekeeper-v3.7.0/config.yml
+time kapp delete -y -a gk
+
+time kapp deploy -y -a pinniped -f examples/pinniped-v0.13.0/
+time kapp delete -y -a pinniped
+
+pkill -9 'minikube'
```

Comment:
Can we add something specific to `tunnel`. I think this might stop `minikube` sometime.

Reply:
It should not, as minikube doesn't run as a process. However, `minikube tunnel` does require password to run, so I am not sure how it would behave in the GitHub Actions environment.

Reply:
that's strange, minikube doesn't run as process. So, you are saying minikube doesn't run as process but minikube tunnel does ?

Reply:
`minikube` runs as a _service_ and `minikube tunnel `runs as a _process_. You can try using `pgrep` to confirm.